tests/functional/modules/test_zos_mount_func.py

```diff
@@ -255,6 +264,75 @@ def test_basic_mount_with_bpx_nocomment_nobackup(ansible_zos_module, volumes_on_
             record_length=80,
         )
 
+def test_basic_mount_with_bpx_no_utf_8_characters_(ansible_zos_module, volumes_on_systems):
```

Comment:
It does not look like the test case is actually testing non UTF-8 characters,  I tested building what we currently have in `dev` branch and then executed this test using :
```bash
./ac --ac-test --host ec33012a --python 3.11 --zoau 1.3.4 --file tests/functional/modules/test_zos_mount_func.py --debug --verbose vvv --test test_basic_mount_with_bpx_no_utf_8_characters_
```